Cargo.toml

```diff
@@ -17,6 +17,7 @@ members = [
     "vm/interpreter",
     "node",
     "node/clock",
+    "node/rpc",
```

Comment:
can you add it to the makefile as one of the local packages to clear for linting?